### ISSUE TITLE
Objects can randomly flicker when using Vec3 as position instead of x,y,z

### ISSUE THREAD

**Issue by brainrom** [bjornbytes/lovr#444]
Now I'm trying to make a short demo example. But synopsis is that when I call a draw function (both model and builtin box, plane, etc.) with Vec3 option instead of coordinates (but Vec3:unpack() helps!) objects can flicker at random moments of time.

**Comment by bjornbytes** [comment]
This is a semi-known issue, though I've had trouble tracking it down.  If you're able to share a small example project that has the issue, that could help getting it fixed.

**Comment by brainrom** [comment]
Okay, I did some research. This example effectively cause repeatable flickering, but loading new texture every update isn't necessary at all. This is just more demonstrative.
```
function lovr.update()
  for i=0,100 do
    tex=lovr.graphics.newTexture(1024,1024,1)
  end
end

function lovr.draw()
  for i=0,100 do
    lovr.graphics.cube('fill', lovr.math.newVec3(1, 1.7, -1*i*1.5))
  end
end

```

**Comment by bjornbytes** [comment]
I was finally able to uncover the cause!  Fixed in 526e06d.  What a relief, thanks for the issue.
